routes/encrypt.ts

```diff
@@ -0,0 +1,22 @@
+const express = require('express')
+const bycrpyt = require('bcryptjs')
+const jwt = require('jsonwebtoken')
+import { use } from "marked"
+import {Db_Wrapper, env_config} from "../config"
+import { encrypt, decrypt } from "../util"
+
+const router = express.Router()
+
+router.post("/encrypt", async (req, res) => {
+    //For debugging purposes. Prints encrypted version of url. In the future, this will be done in researcher menu.
+    const { url, worker_id } = req.body
```

Comment:
This route relies heavily on having `worker_id` but we can imagine some situations where we don't have it. Could we adjust so we don't need to depend on it? 

If that will be really annoying we could accept as is and put that flexibility as an issue for a later cycle.

Reply:
Made it query independent, and only thing required is a url to the survey